releasenotes/notes/fix-nested-propagation-extraction-e3fa2a83b1c21f4c.yaml

```
@@ -0,0 +1,9 @@
+---
+fixes:
+  - |
+    tracing: fixes issues with nested web frameworks re-extracting HTTP context propgation headers.
+
+    This new behavior can be disabled with::
+
+      import ddtrace
+      ddtrace.config.propagation_kip_multiple_extract = False
```

```suggestion
      ddtrace.config.propagation_skip_multiple_extract = False
```

should this feature be this public? I feel like this is a logical new default behaviour of the library and then we provide a hidden config flag to fallback if for some reason we expect a user to need to revert this behaviour (I can't really think of a case though).

---

I think since it changes current behavior it's helpful for users/SEs to see this change is intended and what's going on. I don't see how it could hurt.